src/app/views/App.tsx

```diff
@@ -345,27 +344,27 @@ class App extends Component<IAppProps, IAppState> {
     if (mobileScreen) {
       sidebarWidth = layout = 'col-xs-12 col-sm-12';
     }
-    if (permissionModeType !== PERMISSION_MODE_TYPE.TeamsApp && this.state.hideDialog) {
-      this.showDialog();
-    }
     // eslint-disable-next-line react/jsx-no-target-blank
     const teamsapp = <a href={"https://www.bing.com/?form=000010"} target="_blank">{translateMessage('Sample Explorer Teams app')}</a>;
     //TODO: put in the url when we have this set up ADO #38728
     // eslint-disable-next-line react/jsx-no-target-blank
     const rsc = <a href={RSC_URL} target="_blank">{translateMessage('resource-specific consent')}</a>;
+    // eslint-disable-next-line react/jsx-no-target-blank
+    const installedApp = <a href={INSTALLED_APP_URL} target="_blank">{translateMessage("Installed Apps")}</a>;
     return (
       // @ts-ignore
       <ThemeContext.Provider value={this.props.appTheme}>
         {permissionModeType === PERMISSION_MODE_TYPE.TeamsApp && < Dialog
-          hidden={this.state.hideDialog}
+          hidden={hideDialog}
           dialogContentProps={{
             title: `${translateMessage('Install sample app')}`,
             showCloseButton: true,
           }}
-          onDismiss={this.closeDialog}
+          onDismiss={this.toggleDialog}
         >
           <p>{translateMessage('Resource-specific Consent popup')} {teamsapp}. <br /> &nbsp;</p>
-          <p>{translateMessage('Learn more about')} {rsc}.</p>
+          <p>{translateMessage('Learn more about')} {rsc}. <br /> &nbsp;</p>
+          <p>{translateMessage('Confirm')} {installedApp}.</p>
```

Comment:
Dialogs should have as little actionable items as possible! And the actionable items should be buttons (I'm working on this in another task) https://developer.microsoft.com/en-us/fluentui#/controls/web/dialog

Reply:
Removing the permissions line d84a05f4b2ca6b0f3f2e59b366a968035d2a1e07